larray/core.py

```diff
@@ -3978,7 +3992,7 @@ def replace_axes(self, axes_to_replace=None, new_axis=None, **kwargs):
             items += kwargs.items()
             for old, new in items:
                 axes = axes.replace(old, new)
-        return LArray(self.data, axes, title=self.title)
+        return LArray(self.data, axes)
```

Comment:
We should define a (simple) rule for users to know when attributes are copied and when they are not.